sites/friday/src/get-started/express/_index.md

```diff
@@ -1,4 +1,366 @@
+| App or service        | Size    | CPU     | Memory (MB) | Disk (MB) | Instances |
++-----------------------+---------+---------+-------------+-----------+-----------+
+| app                   | not set | not set | not set     | N/A       | 1         |
++-----------------------+---------+---------+-------------+-----------+-----------+
+```
+The first question is what profile size you want applied to your application image. For now let's select `1`:
+```shell
+App: app
+Choose a profile size:
+  [0.1 ] CPU 0.1, memory 64 MB
+  [0.25] CPU 0.25, memory 128 MB
+  [0.5 ] CPU 0.5, memory 224 MB
+  [1   ] CPU 1, memory 384 MB
+  [2   ] CPU 2, memory 704 MB
+  [4   ] CPU 4, memory 1216 MB
+  [6   ] CPU 6, memory 1728 MB
+  [8   ] CPU 8, memory 2240 MB
+  [10  ] CPU 10, memory 2688 MB
+ > 1
+```
+Next it will ask how many instances of our application container we need deployed. For now let's go with `1`:
+```shell
+Enter the number of instances (default: 1): 1
+```
+
+Last it will ask us to confirm our choices. Select `Y` and the {{% vendor/name %}} will take your selections, grab the
+previous built images from early, apply our resource selections to them and deploy our full application!
+
+## Make changes to your project
+
+Now that your project is deployed, you can start making changes to it.
+For example, you might want to fix a bug or add a new feature.
+
+In your project, the `main` branch always represents the production environment.
+Other branches are for developing new features, fixing bugs, or updating the infrastructure.
+
+To make changes to your project, follow these steps:
+
+1. Create a new environment (a Git branch) to make changes without impacting production:
+
+   ```bash
+   {{% vendor/cli %}} branch feat-a
+   ```
+
+   This command creates a new local `feat-a` Git branch based on the main Git branch
+   and activates a related environment on {{< vendor/name >}}.
+   The new environment inherits the data (service data and assets) of its parent environment (the production environment here).
+
+2. Make changes to your project.
+
+   For example, edit the `./index.js` file and make the following changes:
+
+   ```html {location="index.js"}
+   app.get('/', async function(req, res){
+     // Make the output.
+     const outputString = `Hello, World! - A simple Express web framework template for {{% vendor/name %}}`
+     res.send(outputString);
+   });
+   ```
+
+3. Commit your changes:
+
+   ```bash
+   git commit -a -m "Update Hello world"
+   ```
+
+4. Deploy your changes to the `feat-a` environment:
+
+   ```bash
+   {{% vendor/cli %}} deploy
+   ```
+
+   Note that each environment has its own domain name.
+   To open the url of your new environment, run the following command:
+
+   ```bash
+   {{% vendor/cli %}} environment:url --primary
+   ```
+
+5. Iterate by changing the code, committing, and deploying.
+   When satisfied with your changes, merge them to the main branch, deploy,
+   and remove the feature branch:
```

Comment:
we mention removing the _branch_ but don't mention deactivating/deleting the environment. If they do `git branch -d feat-a` that will leave the remote environment up and running, costing them money. I think if we're going to tell them to delete the branch, we should give them the commands to delete the branch AND deactivate the environment (ie `{vendor} e:delete`)

Reply:
i add more steps into this section. 
So now, we merge, delete+deactivate the env and remove the local git branch